_includes/sidebar.html

```diff
@@ -0,0 +1,132 @@
+<!-- detail page sidebar -->
+{% if page.partner_site %}
+{% assign urlparts = page.url | split: "/" %}
+{% assign year = urlparts[1] %}
+{% assign location = urlparts[2] %}
+{% assign sidebar = site.data[year][location].sidebar %}
+  <div class="col-md-4">
+    <ul class="nav flex-column">
+      <li class="nav-item">
+        <a class="nav-link" href="{{page.dir | prepend: site.baseurl}}">Overview</a>
+      </li>
+      {% for nav in sidebar %}
+        <li class="nav-item">
+          <a class="nav-link" href="{{nav.url}}">{{nav.name}}</a>
+        </li>
+      {% endfor %}
+      <li class="nav-item">
+        <a class="nav-link btn btn-dark d-xl-inline-flex mt-3 ml-3" href="apply">Apply</a>
+      </li>
+      <br/>
+    </ul>
+    <ul class="nav flex-column">
+      <a class="nav-link collapsed" onmouseover=""" style="cursor: pointer;" data-toggle="collapse" data-target="">Other Events:</a>
+      {% assign check2018 = site.data.2018[location].hero %}
```

Comment:
Not the most elegant solution to the "See events from other years" functionality. Any suggestions?

Reply:
Yes. I did something similar here:

https://github.com/compsocialscience/summer-institute/blob/4438440d548c65e2af6852d113d7c94b6f234112/_layouts/all_time_participants.html#L13-L14

You can use that to get a list of all possible years and then check to see if that location existed for each year.

Reply:
thanks! addressed in: https://github.com/compsocialscience/summer-institute/pull/1240/commits/cb3ce84ada63ee46111643ca58e93e6580da1e51